src/components/Profile/ConnectSocials/index.js

```diff
@@ -23,7 +23,7 @@ const ConnectSocials = () => {
         <Box className={classes.row} sx={{ marginBottom: 15 }}>
           <Box
             className={classes.link}
-            onClick={() => signInWithProviderID("facebook")(firebase, dispatch)}
+            // onClick={() => signInWithProviderID("facebook")(firebase, dispatch)}
```

Comment:
why these are commented?

Reply:
They don't connect with the accounts yet. Instead, I have added the inputs for adding a social media handle in the profile section.

Reply:
ah okay